pkg/scraper/client/resource/decode.go

```diff
@@ -145,6 +149,18 @@ func parseContainerMemMetrics(namespaceName apitypes.NamespacedName, containerNa
 	pods[namespaceName].Containers[containerName] = containerMetrics
 }
 
+func parseContainerStartTimeMetrics(namespaceName apitypes.NamespacedName, containerName string, timestamp int64, value float64, pods map[apitypes.NamespacedName]storage.PodMetricsPoint) {
+	if _, findPod := pods[namespaceName]; !findPod {
+		pods[namespaceName] = storage.PodMetricsPoint{Containers: make(map[string]storage.MetricsPoint)}
+	}
+	if _, findContainer := pods[namespaceName].Containers[containerName]; !findContainer {
+		pods[namespaceName].Containers[containerName] = storage.MetricsPoint{}
+	}
+	containerMetrics := pods[namespaceName].Containers[containerName]
+	containerMetrics.StartTime = time.Unix(0, int64(value*1e9))
```

Comment:
we might want to convert the timestamp to nanosecond so that Timestamp and StartTtime have the unit

Reply:
Yeah, butI think we have  converted the timestamp to nanosecond  by ` containerMetrics.Timestamp = time.Unix(0, timestamp*1e6)` .

Reply:
Ah yes my bad, I got confused.